packages/gluestick/src/__tests__/mocks/EntryWrapper.js

```diff
@@ -0,0 +1 @@
+module.exports = () => {};
```

Comment:
These are the minimum to get the tests running. (Likely they'll need to be expanded over time.)